### PR TITLE
Feat/get notifications

### PR DESCRIPTION
**Pull request by ODINKS**

### CODE REVIEW

File: src/controllers/notificationController.ts [hunk 1]
```diff
@@ -14,21 +15,92 @@ export const getNotifications = async (
...
+    // Retrieve notification from notification services
+    const userNotifications = await NotificationService.getNotifications(authUser);
 
-    const notifications = await NotificationService.getNotifications(authUser);
+    // Query the notification database.
+    let query = NotificationModel.find()
 
-    if (notifications == null) {
+
+    if (query == null) {
```
Comment (sudodeo) [inline src/controllers/notificationController.ts:25]: add `|| userNotifications == null` to this if check

File: src/controllers/notificationController.ts [hunk 2]
```diff
@@ -14,21 +15,92 @@ export const getNotifications = async (
...
+    if (!notifications) {
+      throw new ResourceNotFound('resource not found!')
+    }
+
+    const resPayload = {
+      success: true,
       message:
         notifications.length > 0
-          ? "Notifications retrieved successfully"
+          ? "Notification retrieved successfully"
```
Comment (sudodeo) [inline src/controllers/notificationController.ts:84]: Should be 'Notifications' to fit the context of getting all notifications

Comment (ODINKS) [reply src/controllers/notificationController.ts:84]: Alright.

Comment (ODINKS) [reply src/controllers/notificationController.ts]: Done.

**Review by Adedipupo — APPROVED** [review]

**Pull request merged**